src/lamplib/tasks/run_tests.py

```diff
@@ -110,7 +110,11 @@ def resmoke_test(env, suites, mongo_dir, is_cnats):
         # Default download location for MongoDB binaries.
         env["PATH"] += ":" + os.path.join(mongo_dir, "bin") + ":" + mongo_dir
 
-    evg_venv_dir = os.path.join(workdir, "build", "venv")
+    if 'LAMP_VENV_DIR' not in os.environ:
```

Comment:
Do you care about being 'pythonic'? The pythonic way of doing things is, "do it and handle the exception" instead of checking if something will avoid generating an exception in the first place

Reply:
Ah the age old EAFP vs LBYL debate. For this check, I think of it as conceptually belonging to input validation, rather than the more generic theme of error handling. Even in Python, user input validation's main value as I see it is usability, where we can ensure the error message shows up early on to be visible, and we can capture arbitrary information that may exceed the length of an error message.